apps/forklift/lib/forklift/jobs/json_to_orc.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Forklift.Jobs.JsonToOrc do
```

Comment:
So, you're telling me, if we remove 3 layers of pipeline, that this is much easier to read? Woooooow